.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Issue about Taninsam code
+---
+
+> Thanks helping us improving Taninsam
+
+## Main information
+
+> At least, fill those informations
+
+- Version of Taninsam: _x.y.z_
+- List of concerned functions:
+  - _function 1_
+  - _function 2_
+  - _etc..._
+- Current behavior: _add description, please be clear and concise_
+- Expected behavior: _add description, please be clear and concise_
+
+## Reproduction
+
+> To help us further, please reproduction informations
+
+- Code to reproduce:
```

Comment:
It could be nice to suggest to report "Code to reproduce" also with https://npm.runkit.com/taninsam .